src/site/layouts/press_release.drupal.liquid

```diff
@@ -0,0 +1,153 @@
+<div id="content" class="interior">
+    <main class="va-l-detail-page">
+        <div class="usa-grid usa-grid-full">
+            <nav class="usa-width-one-fourth va-sidebarnav" id="va-detailpage-sidebar"></nav>
+            <article class="usa-content">
+                <section class="vads-u-margin-bottom--5">
+                    <h1>{{ title }}</h1>
+                    <p class="vads-u-margin-bottom--1">PRESS RELEASE</p>
+                    <p class="vads-u-font-weight--bold vads-u-margin-bottom--4 vads-u-margin-top--0">{{ fieldReleaseDate.value | formatDate: 'MMMM D, YYYY' }}</p>
+                    <div>
+                        <a href="#" class="vads-u-margin-right--4" onclick="window.print(); return false;"><i class="fa fa-print"></i> Print</a>
+                        {% if fieldPdfVersion != empty %}
+                            <a href="{{ fieldPdfVersion.entity.fieldDocument.entity.url }}" download><i class="fa fa-download"></i> Download a PDF version</a>
+                        {% endif %}
+                    </div>
+                    <p class="va-introtext">{{ fieldAddress.locality }}, {{ fieldAddress.administrativeArea }} — {{ fieldIntroText }}</p>
+                    <div>{{ fieldPressReleaseFulltext.processed }}</div>
+                </section>
+                <section class="vads-u-margin-bottom--5">
+                    <div class="vads-u-font-weight--bold">Media contacts</div>
+                    {% for contact in fieldPressReleaseContact %}
+                        {% assign c = contact.entity %}
+                        <div>
+                            <p>{{ c.title }}{% if c.fieldDescription != empty %}, {{ c.fieldDescription }} {% endif %}</p>
+                            <p>{{ c.fieldPhoneNumber }}</p>
+                            <p><a href="mailto:{{ c.fieldEmailAddress }}">{{ c.fieldEmailAddress }}</a></p>
+                        </div>
+                    {% endfor %}
+                </section>
+                <section class="vads-u-margin-bottom--5">
+                    <div class="vads-u-font-weight--bold vads-u-margin-bottom--1p5">Download media assets</div>
+                    {% for asset in fieldPressReleaseDownloads %}
+                        {% assign a = asset.entity %}
+                        <div class="vads-u-margin-bottom--1">
+                            {% if a.entityBundle === 'document' %}
+                                <a href="{{ a.fieldDocument.entity.url }}" download>{{ a.name }}</a>
+                            {% endif %}
+                            {% if a.entityBundle === 'image' %}
+                                <a href="{{ a.image.url }}" download>{{ a.name }}</a>
+                            {% endif %}
+                            {% if a.entityBundle === 'video' %}
+                                <a href="{{ a.fieldMediaVideoEmbedField }}" target="_blank">{{ a.name }}</a>
+                            {% endif %}
```

Comment:
Just preference, but seems like a [case](https://help.shopify.com/en/themes/liquid/tags/control-flow-tags#case-when) statement could be nice here. 

I'm also wondering if this should render as an unordered list? Or if there are instances where there are no downloads, should we check for `empty` before rendering this section?

Reply:
ok, can do